### ISSUE TITLE
UX - Language Switcher Selected Language as Background Image

### ISSUE THREAD

**Issue by amoradi** [dashpay/dash-website#493]
Selected language (on large screens) is demarcated as a background image. Background images should only be used for decorative purposes. Screen readers cannot read these. 

**Suggestions**
- ditch flags altogether (selected & options) for an exclusively text-based approach
- add text to selected language flag, e.g. "Francais <flag>"
- replace flag image sprite with individual images, so alt attributes can be applied for screen readers

**Comment by chuckwilliams37** [comment]
I agree with all of these suggestions. In the words of Edward Tufte "... integrate word, number and image..."
